src/soft-tabs-transform.coffee

```diff
@@ -0,0 +1,38 @@
+require('es6-shim')
+
+Point = require './point'
+Region = require './region'
+
+module.exports =
+class SoftTabsTransform
+  constructor: (@tabLength) ->
+    @softTab = " ".repeat(@tabLength)
```

Comment:
`softTab` seems like a fine name for this. I was trying to emphasize the distinction between this and a hard tab character, but I think you're right about the "string" suffix being overkill. The **soft** prefix is sufficient to convey the point.